### PR TITLE
added option to watch namespaces

### PR DESCRIPTION
**Pull request by Prajithp**
This PR add an option to watch specific namespaces.

### CODE REVIEW

File: handlers.py [hunk 1]
```diff
@@ -1,9 +1,19 @@
 import kopf
 import kubernetes
-
-
-@kopf.on.create('', 'v1', 'secrets', annotations={'synator/sync': 'yes'})
-@kopf.on.update('', 'v1', 'secrets', annotations={'synator/sync': 'yes'})
+import os
+
+def watch_namespace(namespace, **_):
+    namespaces = os.getenv('WATCH_NAMESPACE')
```
Comment (TheYkk) [inline handlers.py:6]: Could you set this variable out of function? We don't need to get env every time

File: handlers.py [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+    if namespaces is None or namespaces == "":
+        return True
+
+    all_namespaces = namespaces.split(',')
```
Comment (TheYkk) [inline handlers.py:10]: Could you set this variable out of function? We don't need to parse namespaces every time

**Review by TheYkk — CHANGES_REQUESTED** [review]

File: handlers.py (hunk 2 above)
Comment (Prajithp) [reply handlers.py]: Can you please verify?

File: handlers.py (hunk 1 above)
Comment (Prajithp) [reply handlers.py]: Can you please verify?

File: handlers.py [hunk 3]
```diff
@@ -2,16 +2,15 @@
...
-    if namespaces is None or namespaces == "":
-        return True
+WATCH_NAMESPACE = os.getenv('WATCH_NAMESPACE')
+if WATCH_NAMESPACE is not None:
+    all_namespaces = WATCH_NAMESPACE.split(',')
 
-    all_namespaces = namespaces.split(',')
-    if namespace in all_namespaces:
+def watch_namespace(namespace, **_):
+    if WATCH_NAMESPACE == "" or namespace in all_namespaces:
```
Comment (TheYkk) [inline handlers.py:10]: If `WATCH_NAMESPACE` env doesn't exist ; `all_namespaces` will be undefined

Comment (TheYkk) [reply handlers.py:10]: I think we can define `all_namespaces` as empty string

**Review by TheYkk — CHANGES_REQUESTED** [review]

File: handlers.py (hunk 3 above)
Comment (Prajithp) [reply handlers.py:11]: Yeah correct, somehow it got removed when i copy pasted from my test server.

Comment (TheYkk) [reply handlers.py:11]: But again If WATCH_NAMESPACE env doesn't exist ; we can't return true
image

Comment (Prajithp) [reply handlers.py:11]: Ok, in that case if WATCH_NAMESPACE is not set, can we add an  empty string?
```
WATCH_NAMESPACE = os.getenv('WATCH_NAMESPACE', "")
all_namespaces  = WATCH_NAMESPACE.split(',')
```

Comment (TheYkk) [reply handlers.py:11]: Yes we can use this

**Review by TheYkk — APPROVED** [review]
LGTM

**Pull request merged**